src/annotations/reducers/index.test.ts

```diff
@@ -0,0 +1,56 @@
+import {annotationsReducer, initialState} from 'src/annotations/reducers'
+
+const mockAnnotations = [
+  {
+    stream: 'default',
+    annotations: [
+      {
+        summary: 'hi',
+        end: '2021-01-21T00:05:51Z',
```

Comment:
note that the 'end' and 'start' are strings.

the plot is expecting numbers (milliseconds) not date strings.
unless we want to store them like that and do conversions.

besides that the redux looks good.

Reply:
created an issue to address this: #570